app/src/main/java/com/example/splendormobilegame/activities/GameActivity/DeckReservingController.java

```diff
@@ -63,15 +60,20 @@ public class ReservationFromDeckMessageHandler extends UserReaction {
         public UserMessage react(ServerMessage serverMessage) {
             Log.i("UserReaction", "Entered ReservationFromDeckMessageHandler react method");
 
-            
+            // TODO Update the model
+            // TODO Update the view via `gameActivity` or other objects given in constructor
             MakeReservationFromDeck.ResponseData responseData = (MakeReservationFromDeck.ResponseData) ReactionUtils.getResponseData(serverMessage, MakeReservationFromDeck.ResponseData.class);
 
 
             User user=Model.getRoom().getUserByUuid(Model.getUserUuid());
```

Comment:
If we receive this response from the server it doesn't always refer to this current user, it can refer to the other ones. Server send in its message the UUID of the user that reserved the card.